Modules/DevTools/TestFramework/TestLibraries/Any/src/Any.Codeunit.al

```diff
@@ -231,7 +243,7 @@ codeunit 130500 "Any"
     /// <returns>Pseudo-random integer value</returns>
     local procedure GetNextValue(MaxValue: Integer): Integer
     begin
-        if Seed = 0 then
+        if (not SeedSet) then
```

Comment:
We need and Seed = 0 if we want to keep a global Boolean to track if the Seed has not been set, otherwise we risk resetting it too many times. Then this change would allow setting seed to 0, however I'm not sure if that is needed...

Reply:
Not sure I understand. Resetting what? This does not change anything. The behavior is the same as before. The only difference is that you do not always know the actual seed itself, you only know if it has been set or not. Which is all we really need to know.

Setting a Seed to 0 is not really needed, but I also don't think it should be prohibited? Not sure why it would make a difference.